pipelines/broad/arrays/single_sample/Arrays.documentation.md

```diff
@@ -98,28 +169,51 @@ The pipeline executes the following tasks:
 The pipeline generates the following outputs:
 
 *   ChipWellBarcodeOutput. The chip_well barcode (this is the same as the input parameter 'chip_well_barcode').
+
 *   AnalysisVersionNumberOutput. The analysis version number (this is the same as the input parameter 'analysis_version_number').
+
 *   OutputVcfFile. The VCF generated by the pipeline.
+
 *   OutputVcfIndexFile. The index file of the VCF generated by the pipeline.
-*   GTCFile. The GTC file generated by IlluminaGenotypingArray. Autocall
-*   BafRegressMetricsFile.  A metrics file containing the metrics generated by BafRegress
-*   ContaminationMetricsFile. A metrics file containing the metrics generated by VerifyIDIntensity
-*   OutputFingerprintVcfFile. A VCF containing genotypes selected from the output_vcf at certain designated sites
-*   OutputFingerprintVcfIndexFile. The index file of the output_fingerprint_vcf
+
+*   GTCFile. The GTC file generated by IlluminaGenotypingArray Autocall.
```

Comment:
since all of these output variables were refactored to meet python naming conventions, should that be reflected here?